### DISCUSSION TITLE
[Feature]  Image Upload and Similar Image Search Functionality in Immic Project

### DISCUSSION THREAD

**Discussion by neozhu** [immich-app/immich#3041] [Feature Request]
### The feature

I'm excited to propose a new feature for the Immic project that I believe will greatly enhance our image search functionality.

The feature I am suggesting involves the ability for users to upload an image and then search for similar images within the platform. I am aware that implementing this might present a certain level of difficulty, yet I believe the potential benefits far outweigh the challenges.

This feature would be especially useful for searches focused on specific individuals. The ability to find similar images through an uploaded picture could provide a more meaningful search experience than simply using keywords.

I eagerly look forward to hearing your thoughts on this proposal

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2447
